src/SDKs/DeviceProvisioningServices/Management.DeviceProvisioningServices/Microsoft.Azure.Management.DeviceProvisioningServices.csproj

```diff
@@ -8,7 +8,7 @@
     <Description>Provides management capabilities for Microsoft Azure IoT Device Provisioning Service.</Description>
     <AssemblyTitle>Microsoft Azure Provisioning Service Management</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.DeviceProvisioningServices</AssemblyName>
-    <Version>0.9.0-preview</Version>
+    <Version>0.9.0</Version>
```

Comment:
If you are planning to release a stable version, we recommend starting versioning from `1.0.0`

Reply:
Done